pyproject.toml

```
@@ -70,7 +70,7 @@ doc = [
     "sphinx-autobuild==2021.3.14",
     "sphinx-autodoc-typehints==1.23.0", # BLOCKED BY sphinx-design - Cannot upgrade to Sphinx 7 for now
```

```suggestion
    "sphinx-autodoc-typehints==1.24.0",
```

Since upgrade is now possible, updating other packages